src/og/Clock.ts

```diff
@@ -13,39 +27,42 @@ import { Events } from "./Events.js";
  * @param {number} [params.multiplier=1.0] - Time speed multiplier.
  */
 class Clock {
-    static get _staticCounter() {
-        if (!this._counter && this._counter !== 0) {
-            this._counter = 0;
-        }
-        return this._counter;
-    }
+    static __counter__: number;
 
-    static set _staticCounter(n) {
-        this._counter = n;
-    }
+    public __handler: Handler | null = null;
+    public events: EventsHandler<ClockEventsList>;
```

Comment:
the default visibility of a variable in typescript is `public` so there is no need to add it.